flink-core/src/main/java/org/apache/flink/configuration/RestOptions.java

```diff
@@ -193,6 +193,28 @@ public class RestOptions {
                                     + "Lowering the thread priority will give Flink's main components more CPU time whereas "
                                     + "increasing will allocate more time for the REST server's processing.");
 
+    /**
+     * Duration from write after which cached checkpoints statistics are cleaned up. It can be
+     * specified using notation: "500 ms", "1 s".
+     */
+    @Documentation.Section(Documentation.Sections.EXPERT_REST)
+    public static final ConfigOption<Duration> CACHE_CHECKPOINT_STATISTICS_TIMEOUT =
+            key("rest.cache.checkpoint-statistics.timeout")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(3))
+                    .withDescription(
+                            "Duration from write after which cached checkpoints statistics are cleaned up. It can"
+                                    + " be specified using notation: \"500 ms\", \"1 s\".");
```

Comment:
We shouldn't be explaining how to specify duration here. If something needs to be clarified, we should develop a generalized explainer because it applies to many fields.
```suggestion
                    .withDescription(
                            "Duration from write, after which cached checkpoints statistics are cleaned up.");
```

Reply:
Good point. Will incorporate feedback